draft-dr-dprive-tls-and-dtls-authentication.txt

```diff
@@ -141,91 +149,86 @@ Internet-Draft            (D)TLS Authentication            November 2015
    "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
    document are to be interpreted as described in [RFC2119].
 
-   We use several terms specifically in the context of this draft:
+   Several terms are used specifically in the context of this draft:
 
-   o  DNS Client: a DNS stub implementation or forwarding resolver.  A
-      DNS Client receives requests from an application or a system and
-      forwards them to some pre-configured set of upstream DNS
-      Resolvers.  In the case of a forwarding resolver, we use the term
-      "DNS Client" to discuss its active or querying side here.
+   o  DNS client: a DNS stub resolver or forwarder/proxy.  In the case
+      of a forwarder, the term "DNS client" is used to discuss the side
+      that sends queries.
 
-   o  DNS Resolver: a pre-configured upstream resolver that a DNS client
-      talks to.  These are usually recursive resolvers, but may
-      themselves also be forwarding resolvers.  In the case of a
-      forwarding resolver, we use the term "DNS Resolver" to discuss its
-      passive or responding side here.
+   o  DNS server: a DNS recursive resolver or forwarder/proxy.  In the
+      case of a forwarder, the term "DNS server" is used to discuss the
+      side that responds to queries.
 
```

Comment:
I think with regard to the terms here we should either do as above, or use 'DNS stub' and 'DNS Resolver' throughout and clarify that these terms both include the client/server aspects of forwarders/proxies. I don't particularly mind but I think client/server is a more common way to do this?